src/stylesheets/core/_variables.scss

```diff
@@ -406,7 +406,7 @@ $assignments-theme-color: (
   "base-dark": $theme-color-base-dark,
   "base-darker": $theme-color-base-darker,
   "base-darkest": $theme-color-base-darkest,
-  "ink": $theme-color-base-darkest,
+  "ink": $theme-color-base-ink,
```

Comment:
Discovered while debugging [this comment #3906](https://github.com/uswds/uswds/issues/3906#issuecomment-758776179).

Given a custom Ink value
```scss
// Base colors
$theme-color-base-ink: #141414;
```

It was causing the [custom grades](https://github.com/uswds/uswds/pull/3949/files#diff-9bf0d9df140d95fba8158d3f6fd0876a22565f243cf9a2fcae81804304b59eceR1979-R1980) to return `ink-darker`, which doesn't exist.